src/mlagility/analysis/util.py

```diff
@@ -30,10 +30,12 @@ class ModelInfo:
     old_forward: Union[Callable, None] = None
     status_message: str = ""
     status_message_color: printing.Colors = printing.Colors.ENDC
+    traceback_message_color: printing.Colors = printing.Colors.FAIL
     is_target: bool = False
     build_model: bool = False
     model_type: build.ModelType = build.ModelType.PYTORCH
     performance: MeasuredPerformance = None
+    traceback = None
```

Comment:
The return value of `traceback.format_exception` a list of strings ([source](https://docs.python.org/3/library/traceback.html)). Use it as a typing hint here?